src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -748,10 +748,10 @@ private void SearchPackagesAndRefreshUpdateCount(bool useCacheForUpdates)
             .PostOnFailure(nameof(PackageManagerControl), nameof(SearchPackagesAndRefreshUpdateCount));
         }
 
-        // Enable if environment variable RecommendNuGetPackages is set to 1.
+        // Check if user has environment variable of RecommendNuGetPackages set to 1 or is in A/B experiment.
         public bool IsRecommenderFlightEnabled()
         {
-            return _forceRecommender;
+            return _forceRecommender | ExperimentationService.Default.IsCachedFlightEnabled("nugetrecommendpkgs");
```

Comment:
I noticed that 
using Microsoft.VisualStudio.Experimentation;
was already added back in a previous PR. Just curious why it was needed there.

Reply:
doesn't look like it was really needed as part of this change: https://github.com/NuGet/NuGet.Client/commit/4a02c08cb2cfb87c77f0d378e361052944b8f3aa
by @srdjanjovcic